src/index.js

```diff
@@ -76,4 +89,10 @@ webext.startup().then(({ browser }) => {
       }
     }
   });
+  setTimeout(
```

Comment:
Started a comment, but deleted. Looks like this is the TODO "Actually randomly prompt users." So, :+1: